internal/client.go

```diff
@@ -407,19 +407,35 @@ type (
 
 	// ConnectionOptions is provided by SDK consumers to control optional connection params.
 	ConnectionOptions struct {
-		TLS                *tls.Config
+		// TLS configures a connection level security credentials.
+		TLS *tls.Config
+
+		// Authority specifies the value to be used as the :authority pseudo-header.
+		// This value only works only when TLS is nil.
+		Authority string
+
+		// By default, after gRPC connection to server is created, client will make a request to
+		// health check endpoint to make sure that server is accessible.
+		// Set DisableHealthCheck to true to disable it.
 		DisableHealthCheck bool
+
+		// Specify how to long to wait while checking server connection when creating new client.
```

Comment:
Shouldn't the comment start with the Field name?

Reply:
It doesn't have too. But here it is easy to fix.